src/utils/mtev_hash.h

```diff
@@ -38,11 +38,25 @@
 
 #include "mtev_config.h"
 #include <ck_hs.h>
+#include <ck_spinlock.h>
 
 typedef void (*NoitHashFreeFunc)(void *);
 
-typedef struct {
+typedef enum mtev_hash_lock_mode {
+  MTEV_HASH_LOCK_MODE_NONE = 0,
+  MTEV_HASH_LOCK_MODE_MUTEX = 1,
+  MTEV_HASH_LOCK_MODE_SPIN = 2,
+  MTEV_HASH_LOCK_MODE_SIZE
```

Comment:
Actually, this is not needed, it was there for an earlier approach where lock functions we stored in an array.  I will remove.